.isort.cfg

```diff
@@ -0,0 +1,19 @@
+[settings]
+line_length = 99
+use_parentheses = True
+include_trailing_comma = True
+multi_line_output=  3
+indent = '    '
+combine_as_imports = True
+case_sensitive = True
+order_by_type = True
+lines_between_types = 1
+force_grid_wrap = 0
+sections = FUTURE,STDLIB,THIRDPARTY,FIRSTPARTY,LOCALFOLDER
+default_section = THIRDPARTY
+import_heading_stdlib = Standard Library
```

Comment:
All of these headings are just visual noise. If isort is being enforced, they don't do anything useful, isort will sort it out anyway.

Reply:
So yeah this came up on discord before, yes they are noise but ... it helps people who are less familiar with the ordering  prior to running isort. So this is more educational than anything happy to remove but  from the discord talks they would be here to explain the ordering so people  learn  it rather than just having a tools "randomly" sort things priort to commiting.

Reply:
Info like this belongs in contributing.md, not in every source code file. This isn't a critical comment that needs visibility everywhere something is in use.